openapi/code_samples/python/api_v1_organizations_{organization_id}_clients_{client_id}/patch.py

```
@@ -1,19 +1,19 @@
-    description="Service account for GitHub Actions to deploy applications
-to production_eu",
+    description="Service account for GitHub Actions to deploy applications to production_eu",
     custom_claims=[
         {"key": "github_repository", "value": "acmecorp/inventory"},
-        {"key": "environment", "value": "production_eu"}
-    ]
+        {"key": "environment", "value": "production_eu"},
+    ],
 )
 
 response = scalekit_client.m2m_client.update_organization_client(
```

Ruff (0.15.21)

[error] Undefined name `scalekit_client`

(F821)